contracts/FundsSplitter.sol

```diff
@@ -57,10 +55,9 @@ contract FundsSplitter {
 
     /**
      * @dev core fund splitting functionality as part of the funds are sent to client and part to starbase
-     * @param value Eth amount to be split
      */
-    function splitFunds(uint256 value) internal {
-        uint256 starbaseShare = value.mul(starbasePercentage).div(100);
+    function splitFunds() public payable {
+        uint256 starbaseShare = msg.value.mul(starbasePercentage).div(100);
```

Comment:
This line here won't necessarily find the starbaseShare. For example the call to the function can be without a message value. 

It needs to be taken from the contract balance of ether

Reply:
When using the TokenSale, it would not be 0 though, right? Are you thinking of a specific scenario?